src/core/js/router.js

```diff
@@ -31,11 +31,21 @@ define([
 
         handleRoute: function() {
             var args = [].slice.call(arguments, 0, arguments.length);
```

Comment:
I know you didn't touch this line, but wondered if you thought 
```
var args = _.toArray(arguments);
```
would look neater?